tests/test_topics.py

```diff
@@ -0,0 +1,180 @@
+import networkx as nx
+import numpy as np
+import pytest
+from gensim.models import KeyedVectors
+from gensim.scripts.glove2word2vec import glove2word2vec
+from nltk.corpus import wordnet as wn
+
+from extra_model._topics import (
+    aggregate,
+    filter_aggregates,
+    get_nodevec,
+    has_connection,
+    iterate,
+    path_to_graph,
+    traverse_tree,
+)
+from extra_model._vectorizer import Vectorizer
+
+
+@pytest.fixture()
+def minivec():
+    # preprocess plain-text test embeddings to proper binary format for vectorizer
+    glove_file = "tests/resources/test_topics.vec"
+    tmp_file = "tests/resources/test_topics.tmp"
+    _ = glove2word2vec(glove_file, tmp_file)
+    model = KeyedVectors.load_word2vec_format(tmp_file)
+    model.save("tests/resources/test_topics.prepro")
+
+    return Vectorizer("tests/resources/test_topics.prepro")
+
+
+@pytest.fixture()
+def vec():
+    return Vectorizer("tests/resources/small_embeddings")
+
+
+@pytest.fixture()
+def simple_graph():
+    graph = nx.DiGraph()
+    graph.add_nodes_from(["L1", "I1", "L2", "I2", "R"])
```

Comment:
would be helpful to have a comment here about what this graph is supposed to represent.